### PR TITLE
add karmada failover example

### PR DESCRIPTION
**Pull request by JadeFlute0127**
Signed-off-by: [JadeFlute0127] [1635468471@qq.com]

**What this PR does / why we need it:**
add karmada failover example

**Which issue(s) this PR fixes:**
Part of #1 

**Special notes for your reviewer:**

**Does this PR introduce a user-facing change?:**

```
NONE

### CODE REVIEW

**Comment by JadeFlute0127** [comment]
Current effect demo website：
https://killercoda.com/zhangdiandian/scenario/karmada-Failover-example

File: karmada-Failover-example/step2/text.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### create deployment
+
+create deployment named nginx
+
+   RUN `kubectl --kubeconfig /etc/karmada/karmada-apiserver.config apply -f ~/nginx/nginxDeployment.yaml`{{exec}}
```
Comment (jwcesign) [inline karmada-Failover-example/step2/text.md:5]: Please add the `cat` command to display the details of the YAML file.

File: karmada-Failover-example/step4/text.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### Check the status and quantity distribution of pods and deployments
+
+RUN `karmadactl --kubeconfig /etc/karmada/karmada-apiserver.config get deployment `{{exec}}
```
Comment (jwcesign) [inline karmada-Failover-example/step4/text.md:3]: I think directly using `karmadactl get pods xxxx --watch` is a better idea, users could see the migration.

Comment (JadeFlute0127) [reply karmada-Failover-example/step4/text.md]: you're right

File: karmada-Failover-example/step2/text.md (hunk 1 above)
Comment (JadeFlute0127) [reply karmada-Failover-example/step2/text.md:9]: fixed

File: karmada-Failover-example/finish.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+**Summary**
+
+In this scenario, we learned how to achieve failover in karmada by simulating faults.
```
[the remark this answers is not in the document]
Comment (JadeFlute0127) [reply karmada-Failover-example/finish.md]: fixed

File: karmada-Failover-example/index.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+          "text": "step1/text.md",
+          "verify": "step1/verify.sh"
+        },
+        {
+          "title": "Check workloads has been deployed",
+          "text": "step2/text.md",
+          "verify": "step2/verify.sh"
+        },
+        {
+          "title": "Simulate taint",
```
[the remark this answers is not in the document]
Comment (JadeFlute0127) [reply karmada-Failover-example/index.json]: fixed

File: karmada-Failover-example/intro.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# What is Karmada?
+
+Karmada (Kubernetes Armada) is a Kubernetes management system that enables you to run your cloud-native applications across multiple Kubernetes clusters and clouds, with no changes to your applications. By speaking Kubernetes-native APIs and providing advanced scheduling capabilities, Karmada enables truly open, multi-cloud Kubernetes.
+
+Karmada aims to provide turnkey automation for multi-cluster application management in multi-cloud and hybrid cloud scenarios, with key features such as centralized multi-cloud management, high availability, failure recovery, and traffic scheduling.
+
+In this scenario, we will learn how to achieve failover in karmada by simulating faults.
```
[the remark this answers is not in the document]
Comment (JadeFlute0127) [reply karmada-Failover-example/intro.md]: fixed

File: karmada-Failover-example/index.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{
+    "title": "karmada failover example",
```
Comment (jwcesign) [inline karmada-Failover-example/index.json:2]: Karmada

File: karmada-Failover-example/intro.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# What is Karmada?
+
+Karmada (Kubernetes Armada) is a Kubernetes management system that enables you to run your cloud-native applications across multiple Kubernetes clusters and clouds, with no changes to your applications. By speaking Kubernetes-native APIs and providing advanced scheduling capabilities, Karmada enables truly open, multi-cloud Kubernetes.
+
+Karmada aims to provide turnkey automation for multi-cluster application management in multi-cloud and hybrid cloud scenarios, with key features such as centralized multi-cloud management, high availability, failure recovery, and traffic scheduling.
+
+In this scenario, we will learn how to achieve failover in karmada by simulating faults using taints.
```
Comment (jwcesign) [inline karmada-Failover-example/intro.md:7]: Karmada, please check other places and ensure that the first letter is capitalized.

File: karmada-Failover-example/step3/text.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
+### Simulate taint
+
+RUN `karmadactl --kubeconfig /etc/karmada/karmada-apiserver.config taint cluster kind-member2 hello=:NoExecute`{{exec}}
```
Comment (jwcesign) [inline karmada-Failover-example/step3/text.md:3]: `failover-test=:NoExecute` is a better name?

**Review by jwcesign — COMMENTED** [review]
other lgtm

File: karmada-Failover-example/intro.md (hunk 7 above)
Comment (JadeFlute0127) [reply karmada-Failover-example/intro.md]: I confirm that the first letter of the karmada string in all documents is already capitalized~

File: karmada-Failover-example/step3/text.md (hunk 8 above)
Comment (JadeFlute0127) [reply karmada-Failover-example/step3/text.md]: yeah , fixed it

File: karmada-Failover-example/index.json (hunk 6 above)
Comment (JadeFlute0127) [reply karmada-Failover-example/index.json]: fixed

File: karmada-Failover-example/step2/text.md (hunk 1 above)
Comment (jwcesign) [reply karmada-Failover-example/step2/text.md:9]: also `cat` the PropagationPolicy file.

File: karmada-Failover-example/step4/text.md [hunk 9]
```diff
@@ -0,0 +1,3 @@
+### Check the status and quantity distribution of pods
+
+RUN `karmadactl --kubeconfig /etc/karmada/karmada-apiserver.config get pods nginx --watch`{{exec}}
```
Comment (jwcesign) [inline karmada-Failover-example/step4/text.md:3]: RUN `karmadactl --kubeconfig /etc/karmada/karmada-apiserver.config get pods --watch`{{exec}}

**Comment by jwcesign** [comment]
/lgtm
cc @RainbowMango for checking

**Pull request merged**